File: portal/portal/App_Code/Midia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace portal.App_Code
{
    public class Midia
    {
        private double _codigo;
        private string _nome;
        private string _url;
        private string _data;

        public double Codigo
        {
            get
            {
                return _codigo;
            }

            set
            {
                _codigo = value;
            }
        }

        public string Nome
        {
            get
            {
                return _nome;
            }

            set
            {
                _nome = value;
            }
        }

        public string Url
        {
            get
            {
                return _url;
            }

            set
            {
                _url = value;
            }
        }

        public string Data
        {
            get
            {
                return _data;
            }

            set
            {
                _data = value;
            }
        }
    }
}

File: portal/portal/App_Code/Midia.php
<?php
###################################################################################################################
/*MÍDIA: Gerencia Arquivos que estão feitos Uploads.
EX: POWER POINT, DOC, EXCEL, PPT, ETC SEAO OS ADMS QUE FAZEM ESSES UPLOADS.*/
####################################################################################################################

namespace portal\App_Code {
    class Midia {
//ATRIBUTOS----------------------------------------------------------------------------------------------------------				
        private $codigo;
        private $nome;
        private $url;
        private $data;
//CONSTRUTOR----------------------------------------------------------------------------------------------------------			
		public function __construct (){}
        
        public function setMidia($codigo,$nome,$URL,$data){
			$this -> codigo = $codigo;
			$this -> nome = $nome;
			$this -> url = $URL;
			$this -> data = $data;
		}
//SETS-----------------------------------------------------------------------------------------------------------------        
		public function setCodigo($codigo){
			$this -> codigo = $codigo;
        }
        public function setNome($nome) {
			$this -> nome = $nome;
        }
		public function setURL($URL) {
			$this -> url = $URL;
        }
		public function setData($data) {
			$this -> data = $data;
        }
//GETS-----------------------------------------------------------------------------------------------------------------				
		public function getCodigo(){
			return $this -> codigo;
        }
        public function getNome() {
			return $this -> nome;
        }
		public function getURL() {
			return $this -> url;
        }
		public function getData() {
			return $this -> data;
        }	
    }
}
?>